### DISCUSSION TITLE
How to enable additional Dictionaries for spell checking on Collabora

### DISCUSSION THREAD

**Discussion by cerebrux** [nextcloud/all-in-one#761] [Questions] [answered]
Is there a way to enable additional dictionaries before or after the installation of Nextcloud AIO with Collabora ? 
Particularly I am interested in enabling Greek (el_GR) spell checking.

I have been able toy enable additional dictionaries on the official colabora/code image by passing -e ‘dictionaries=eu eu_ES eu_FR fr_FR en_GB es_ES el_GR’ on the docker command as well as in a docker-compose file but I would like to be able to do it also with AIO.
Is there an environment variable exposed that I can use to pass dictionaries ?

**Comment by szaimen** [comment]
If there is no way from the collabora interface to enable spell checking for other languages, we will work on a way to set this environmental variable...

**Reply by cerebrux** [reply]
Thanks for your reply. For the time being I use OnlyOffice option of the AIO, on x86_64 which has Greek Spelling out-of-the box. 
But it would be nice to switch to Collabora as I have a personal Arm64 cloud server and OnlyOffice is not compatible.

**Reply by szaimen** [reply]
Onlyoffice will be supported on arm64 as soon as ascencio provides multiarch images but we will provide a way to enable additional spell checking an some point.

**Comment by szaimen** [comment]
Will be addressed with https://github.com/nextcloud/all-in-one/pull/865

**Comment by dantefromhell** [comment]
Sorry to high-jack the topic, but starting new discussions is disabled and this topic is the closest.

Would it also be possible to allow adding additional fonts to Collabora?
https://help.nextcloud.com/t/collobora-online-and-docker-how-to-add-new-fonts/65750/13
Describes how this can be achieved by mounting volume into the collbora docker container

**Reply by szaimen** [reply]
Please open a new discussion for this!

**Reply by dantefromhell** [reply]
As said above, it's currently not possible to start new discussions.
My experience of the discussions tab matches this issue
https://github.com/github-community/community/discussions/29

According to the answer there is a configuration option that disables creation of new discussions while still allowing to answer existing discussions

**Reply by szaimen** [reply]
Please try again now!

**Comment by szaimen** [comment, marked as answer]
Hello, this was just released with 1.5.0 Beta. See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
